### PR TITLE
Feat/add css handles

### PR DESCRIPTION
**Pull request by kaisermann**
#trivial 
#### What is the purpose of this pull request?

Add css handles to ease the layout customization.

#### How should this be manually tested?

1) Go to https://kiwi--storecomponents.myvtex.com/checkout/orderPlaced/?og=984210525286
2) check the markup for the css classes

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: store/interfaces.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "allowed": "*"
   },
   "order-title": {
-    "component": "OrderTitle"
+    "component": "OrderNumber"
```
Comment (salesfelipe) [inline store/interfaces.json]: Why did you rename it? If you are really going to rename it you should also change the test name file and this interface.

Comment (kaisermann) [reply store/interfaces.json]: Oooops, nice catch!

Comment (kaisermann) [reply store/interfaces.json]: Done

File: react/OrderSection.tsx [hunk 2]
```diff
@@ -1,12 +1,31 @@
 import React, { FC } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
 
 interface Props {
   borderless?: boolean
+  className?: string
+  id: string
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/OrderSection.tsx]: Would `name` be ok?

File: react/OrderPlacedConfirmation.tsx [hunk 3]
```diff
@@ -15,7 +15,10 @@ const OrderPlacedConfirmation: FC = () => {
   const hasTakeAway = orderGroup.totalTakeAwayParcels.length > 0
 
   return (
-    <OrderSection borderless={(hasDelivery || hasPickUp) && !hasTakeAway}>
+    <OrderSection
```
[the remark this answers is not in the document]
Comment (salesfelipe) [reply react/OrderPlacedConfirmation.tsx]: Dunno, I think the one inside the OrderSection should do it

File: react/DeliveryItems.tsx [hunk 4]
```diff
@@ -1,25 +1,32 @@
 import React, { FC } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
 
 import ProductList from './components/ProductList'
 import DeliveryHeader from './components/DeliveryHeader'
 import { useOrder } from './components/OrderContext'
 import OrderSection from './OrderSection'
 
+const CSS_HANDLES = ['parcel', 'parcelHeaderColumn']
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/DeliveryItems.tsx]: `orderSection` is already used. I've renamed it to `package` as discusses IRL

File: react/OrderPlacedConfirmation.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/OrderPlacedConfirmation.tsx]: Done 🤙

File: react/PickupItems.tsx [hunk 5]
```diff
@@ -1,25 +1,32 @@
 import React, { FC } from 'react'
+import { applyModifiers, useCssHandles } from 'vtex.css-handles'
 
 import { useOrder } from './components/OrderContext'
 import ProductList from './components/ProductList'
 import StorePickUpHeader from './components/PickUpHeader'
 import OrderSection from './OrderSection'
 
+const CSS_HANDLES = ['parcel', 'parcelHeaderColumn']
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/PickupItems.tsx]: Done 🤙

File: react/components/DeliveryHeader.tsx [hunk 6]
```diff
@@ -2,6 +2,15 @@ import React, { FC, Fragment } from 'react'
 import { FormattedMessage } from 'react-intl'
 import { TranslateEstimate } from 'vtex.shipping-estimate-translator'
 import { Address } from 'vtex.order-details'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+
+const CSS_HANDLES = [
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/components/DeliveryHeader.tsx]: Done 🤙

File: react/components/ErrorMessage.tsx [hunk 7]
```diff
@@ -11,15 +12,16 @@ interface Props {
 }
 
 const ErrorMessage: FC<Props> = ({ icon, errorId, messageId, children }) => {
+  const handles = useCssHandles(CSS_HANDLES)
   return (
-    <div className={styles.errorWrapper}>
+    <div className={handles.errorWrapper}>
       <main>
         {icon}
         <h2 className="f2">
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/components/ErrorMessage.tsx]: Doooone

File: react/components/PickUpHeader.tsx [hunk 8]
```diff
@@ -2,6 +2,16 @@ import React, { FC, Fragment } from 'react'
...
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+
+const CSS_HANDLES = [
+  'parcelHeader',
+  'parcelShippingEstimate',
+  'parcelInfoWrapper',
+  'parcelAddressWrapper',
+  'parcelReceiver',
+  'parcelAdditionalInfo',
+]
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/components/PickUpHeader.tsx]: Doooooone

File: react/components/PickUpHeader.tsx [hunk 9]
```diff
@@ -40,13 +53,17 @@ const StorePickUpHeader: FC<Props> = ({ shippingData, index, numPackages }) => {
...
-        <div className="mb5 mr10-m">
+      <div
+        className={`${handles.parcelInfoWrapper} flex justify-left-m flex-column-s flex-column-l flex-row-m justify-between-l justify-start-l`}
+      >
+        <div className={`${handles.parcelAddressWrapper} mb5 mr10-m`}>
           <Address address={shippingData.address} pickup={shippingData} />
         </div>
-        <div className="c-on-base lh-copy">
+        <div className={`${handles.parcelReceiver} c-on-base lh-copy`}>
           <p>{receiverName}</p>
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/components/PickUpHeader.tsx]: They would be able to style it via the `packageReceiver` handle!

File: react/components/ProductList/Subscription.tsx [hunk 10]
```diff
@@ -1,14 +1,27 @@
...
+  'attachmentContent',
+]
+
 interface Props {
   attachmentItem: Attachment
 }
 
 const SubscriptionAttachment: FC<Props> = ({ attachmentItem }) => {
+  const handles = useCssHandles(CSS_HANDLES)
+  handles
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/components/ProductList/Subscription.tsx]: Already removed 😆

File: react/components/BankInvoice/BarCode.tsx [hunk 11]
```diff
@@ -2,33 +2,37 @@ import React, { FC } from 'react'
...
-    data-testid="bank-invoice-barcode"
-    className="flex-l b--muted-4 ba br3 bw1"
-  >
+const BarCode: FC<Props> = ({ barCodeNumber }) => {
+  const handles = useCssHandles(CSS_HANDLES)
+  return (
     <div
-      className={`${styles.barCodeNumber} tc ph7 ph9-m pv4 lh-copy c-on-base`}
+      data-testid="bank-invoice-barcode"
+      className={`${handles.barCodeWrapper} flex-l b--muted-4 ba br3 bw1`}
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply react/components/BankInvoice/BarCode.tsx]: The `barCodeNumber` handle is used on line #L21 😁
